<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewObject</class>
 <widget class="QWidget" name="NewObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="GroupColumns">
     <item>
      <layout class="QVBoxLayout" name="Column1">
       <item>
        <widget class="QGroupBox" name="GeneralForm">
         <property name="title">
          <string>Général</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nom commun</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="NameLineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Messier</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="MessierSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>110</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>NGC</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="NgcSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>7840</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Autre nom</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="OtherNameLineEdit_1"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="TypeComboBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Constellation</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="ConstellationComboBox_2"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Magnitude apparente</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="ApparentMagnitudeDoubleSpinBox_1">
            <property name="minimum">
             <double>-30.000000000000000</double>
            </property>
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Magnitude apparente secondaire</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QDoubleSpinBox" name="ApparentMagnitudeDoubleSpinBox_2">
            <property name="minimum">
             <double>-30.000000000000000</double>
            </property>
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QDoubleSpinBox" name="DistanceDoubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Diamètre</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QDoubleSpinBox" name="DiametreDoubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="OtherNameLineEdit_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Ascention droite</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="spinBox_3">
              <property name="maximum">
               <number>23</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>h</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_4">
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>m</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
              <property name="maximum">
               <double>59.990000000000002</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Déclinaison</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSpinBox" name="spinBox_5">
              <property name="maximum">
               <number>359</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>°</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_6">
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>'</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
              <property name="maximum">
               <double>59.990000000000002</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>&quot;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Cartes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Niveau 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Niveau 1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Niveau 3</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="spinBox_7">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_8">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="spinBox_9">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Calculer automatiquement à partir de la position</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Appréciation</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Note</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSlider">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
